### PR TITLE
Update Repo

### LINKED ISSUE #1910: chore: move `ci-base-image` publishing out of `merge.yml` and into its own branch
Changing `ci-base-image` requires a 2-step process, such that in step 1 the image is published and in step 2 an additional PR is required to update any github workflows that have changed.

Moving publishing to its own branch de-couples this 2-step process and allows for more flexibility in maintenance.

Note: the second PR usually involves the `nightly-YYYY-MM-DD` changing. If `nightly` dependence is removed, the second PR may not be necessary because the other dependence on `ci-base-image` is set to `latest`, which gets updated with each new `ci-base-image` version. See #1909 for tracking state of `nightly` dependence.

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to update the GitHub org of the repository

# Discussion
- Updated `LibertyDSNP` -> `frequency-chain`
- Move the CI image build to branch [`ci/ci-base-image`](https://github.com/frequency-chain/frequency/tree/ci/ci-base-image) Closes #1910
- Updated Copyright names

### CODE REVIEW

**Review by JoeCap08055 — COMMENTED** [review]
Besides org name changes and linting, I see some removed logic related to the ci-base-image. Could you comment on the purpose of that in relation to the org move?

**Comment by wilwade** [comment]
> Besides org name changes and linting, I see some removed logic related to the ci-base-image. Could you comment on the purpose of that in relation to the org move?

@JoeCap08055 It was noted in the PR where it went, but the reason why is:

1. The container registry images didn't move over, so we had to publish the images before being able to run any PRs
2. We had wanted to get out of the double PR requirement and disconnect the image publishing from the normal CI flow.

File: .github/workflows/merge-pr.yml [hunk 1]
```diff
@@ -74,53 +58,3 @@ jobs:
         uses: ./.github/workflows/common/codecov
         with:
           code-cov-token: ${{ secrets.CODECOV_TOKEN }}
-
-  publish-ci-base-image:
```
Comment (wilwade) [inline .github/workflows/merge-pr.yml:78]: All of this is moved into a separate branch: `ci/ci-base-image`

File: README.md [hunk 2]
```diff
@@ -251,7 +251,7 @@ This option runs one collator node as local host process and two relay chain val
 
 ### All in Docker Container
 
-:exclamation: Currently does not work on M\* series MacOS laptops. See https://github.com/LibertyDSNP/frequency/issues/432
+:exclamation: Currently does not work on M\* series MacOS laptops. See https://github.com/frequency-chain/frequency/issues/432
```
Comment (mattheworris) [inline README.md:254]: 432 was closed, this is probably the best tracking option now.
```suggestion
:exclamation: Currently does not work on M\* series MacOS laptops. See https://github.com/frequency-chain/frequency/issues/779
```

**Review by mattheworris — APPROVED** [review]
- Read through all changes (skimmed *.md changes)

🚢 it!

**Review by JoeCap08055 — APPROVED** [review]
Pipeline is green, so 🚢 it

**Review by saraswatpuneet — APPROVED** [review]
Changes look legit
* Just compiled locally so all good

**Pull request merged**